src/clims/plugins/demo/handlers/create_example_data.py

```diff
@@ -18,6 +20,14 @@ def handle(self):
         Any plugin that implements this handler will supply demo data
         """
 
+        # In case create example data is run on its own, (not within make fresh)
+        # the plugin registration is needed
+        demo_plugin_full_name = '{}.{}'.format(DemoPlugin.__module__, DemoPlugin.__name__)
```

Comment:
I think the reason it works when you're running `make fresh` is that `make fresh` also runs `lims upgrade`. The idea is that when devs want new plugin code they must do two things: (1) pip install the plugin package (2) run `lims upgrade`. This registers all new types.

Would you mind removing this code and see if it works to just run `lims upgrade` before creating demo data?

Reply:
This is a test comment. Checking if our slack integration picks it up!